File: C/PRAK403-2510817210005-Daffa-Dziibaan-Khoiri.c
#include <stdio.h>
#include <stdlib.h>

int main(){

    int input1,input2,angka1, angka2;

    scanf("%d %d", &input1, &input2);

    angka1 = input1;
    angka2 = input2;
    int berhenti = abs(input1 - input2) + 1;
    for(int i=0; i < berhenti; i++){
        
        if(input1 > input2){
            printf("%d %d", angka1, angka2);
            if (i < berhenti - 1)
            {
                printf(" - ");
            }
            
            angka1--;
            angka2++;
        }else{
            printf("%d %d", angka1, angka2);
             if (i <     berhenti - 1)
            {
                printf(" - ");
            }
            angka1++;
            angka2--;
        }      
    }
    return 0;
}

File: Python/PRAK403-2510817210005-Daffa-Dziibaan-Khoiri.py
input1, input2 = map(int, input("").strip().split())

angka1 = input1
angka2 = input2
berhenti = abs(input1 - input2) + 1

for i in range(0, berhenti, 1):
    if input1 > input2 :
        print(f"{angka1} {angka2}", end=" ")
        if i < berhenti- 1 :
            print(" - ", end="")

        angka1 -= 1
        angka2 += 1
    else :
        print(f"{angka1} {angka2}", end="")
        if i < berhenti - 1 :
            print(" - ", end="")

        angka1 += 1
        angka2 -= 1
